core/src/qos_service.rs

```diff
@@ -177,24 +177,22 @@ impl QosService {
     pub fn update_or_remove_transaction_costs<'a>(
         transaction_costs: impl Iterator<Item = &'a TransactionCost>,
         transaction_qos_results: impl Iterator<Item = &'a transaction::Result<()>>,
-        retryable_transaction_indexes: &[usize],
+        transaction_executed_status: impl Iterator<Item = &'a bool>,
```

Comment:
nit: rename to `transaction_commitment_status`

Reply:
Ah that's what I meant, thanks!

Reply:
✅